infocmdb/v1/cmdb/cmdb_test.go

```diff
@@ -28,13 +28,11 @@ CmdbBasePath: /app/`)
 )
 
 func init() {
-	if err := godotenv.Load("../../.env"); err != nil {
-		log.Fatalf("failed to load env: %v", err)
-	}
-
 	if os.Getenv("WORKFLOW_TEST_MOCKING") == "true" {
 		mocking = true
 		log.Debug("Mocking enabled")
+	} else if err := godotenv.Load("../../.env"); err != nil {
+		log.Fatalf("failed to load env: %v", err)
 	}
```

Comment:
the environment should be loaded in any case, if the file exists.

maybe allow the env-file to be given via an environment-variable.

Reply:
* the environment should be loaded in any case, if the file exists.
I am going to change this.
* maybe allow the env-file to be given via an environment-variable.
I'd like to not do this here yet. Could be done in another pull request.